packages/core/src/messages/responses/cache-dictionary-get-field.ts

```diff
@@ -29,14 +29,30 @@ const TEXT_DECODER = new TextDecoder();
  * }
  * ```
  */
-export abstract class Response extends ResponseBase {}
+export abstract class Response extends ResponseBase {
+  public value(): string | undefined {
+    if (this instanceof Hit) {
+      return (this as Hit).value();
+    }
+    return undefined;
+  }
+}
 
 class _Hit extends Response {
   private readonly body: Uint8Array;
   constructor(body: Uint8Array) {
     super();
     this.body = body;
   }
+
+  /**
+   * Returns the data as a utf-8 string, decoded from the underlying byte array.
+   * @returns string
+   */
+  public value(): string {
+    return TEXT_DECODER.decode(this.body);
```

Comment:
Why duplicate the logic in some responses and call the pre-existing value functions in others?

Reply:
very good callout, will fix

Reply:
addressed in https://github.com/momentohq/client-sdk-javascript/pull/760/commits/383af5f8b5f34a70ce4a0cf6efa6bd440dd70fb6